.env

```diff
@@ -48,7 +48,7 @@ EARTHCHEM_DELETE_URL=https://staging-ecl-api.earthchem.org/v2/records/%s
 EARTHCHEM_FILE_CREATE_URL=https://staging-ecl-api.earthchem.org/v2/files/%s/uploadFile
 EARTHCHEM_FILE_DELETE_URL=https://staging-ecl-api.earthchem.org/v2/files/%s/%s
 EARTHCHEM_FILE_READ_URL=https://staging-ecl-api.earthchem.org/v2/files/%s
-EARTHCHEM_VIEW_URL=https://test-ecl.earthchem.org/view.php?id=%s
+EARTHCHEM_VIEW_URL=https://test-ecl.earthchem.org/my/my_view.php?id=%s
```

Comment:
Note that I've updated this URL in the environment file and will probably need to be applied to the environment files of other deployments.

Reply:
Updated this in all environments